pkgs/development/libraries/sqlitecpp/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib, stdenv, fetchFromGitHub, cmake, sqlite, cppcheck, gtest }:
+
+stdenv.mkDerivation rec {
+  pname = "SQLiteCpp";
```

Comment:
pname should match the attribute name.

Reply:
@SuperSandro2000 thanks for the review. I noticed you comment this a lot. Is it a convention agreed upon in an RFC somewhere?

Reply:
It isn't since some wrappers and other things add something additional to the attr name but thats not the case here and it is only confusing when the package shows up as something else in the searches or env.